### PR TITLE
Catalan translation corrections

### PR DESCRIPTION
**Pull request by RetroAND**
Mostly fixing typos on most files and on the script until line #995 (Catalan translation).

### CODE REVIEW

**Comment by maxim-zhao** [comment]
@RetroAND Should I wait for more changes to add to the PR?
@kusfo Does it look good to you?

**Comment by RetroAND** [comment]
Hello,

@maxim-zhao 
Today I will check the rest of the script.
My approach at this time is cautious - just fixing typos, ortographic and syntax errors, etc.

When this first phase is done, I would like to talk about rewording some parts, and to adapt some of the core words of the game (names, etc.) to comply with our phonetic rules so they can be pronounced like the original word was.

I have neither played the game nor own a copy of it, therefore I have no legal way to test those modifications. If you are wondering why went head-on with the corrections without those requisites, is because when I saw that tiles displaying "botiga" I felt projects like this are invaluable to my language.

I am used to git - but not by working in a group. If there is any thing I should change/polish, please tell me and will try to adapt with some time.

**Comment by maxim-zhao** [comment]
You seem to be working GitHub pull requests perfectly well 😃 

I suggest if possible to join our Discord chat at https://discord.gg/uUvF2wbCDX

You should be able to compile it locally if you are able to use some Windows tools, I have not yet tried to make it build on Linux or Mac. There is a makefile and a tools package at https://github.com/maxim-zhao/sms-build-tools, plus you need to add the Japanese game ROM which is not free to distribute. Then you can test in an emulator, although I guess it is hard to test this way if you are not familiar with the game, it is quite difficult and long...

I'll have a check on the changes and maybe merge them to a language-specific branch while the work is in progress. @kusfo is the original Catalan contributor so he's welcome to get involved in the discussion.

**Comment by maxim-zhao** [comment]
I'd advise to compare each script entry to the "literal" and "en" scripts to clarify any confusion about meaning. The goal of the translation is not to be literal, so some deviation/lengthening is OK if it makes the script better to read.

**Comment by RetroAND** [comment]
> I'd advise to compare each script entry to the "literal" and "en" scripts to clarify any confusion about meaning. The goal of the translation is not to be literal, so some deviation/lengthening is OK if it makes the script better to read.

Nice. Then when I finish this first revision, a rewording will be made.

**Comment by RetroAND** [comment]
My second revision (with rewording) is done.

File: psrp/ca/script.yaml [hunk 1]
```diff
@@ -1374,64 +1374,64 @@
...
 
 - offsets: 52a0
   # Honest Dezorian
-  ca: No en volem saber re, dels de Palma!.<wait>
+  ca: No en volem saber res, dels de Palma!.<wait>
 
 - offsets: 52a6
   # Dishonest Dezorian
-  ca: Sa Font de sa Joventut és dintre sa Torre de sa Corona. T'ho juro!<wait>
+  ca: La Font de la Joventut és dis la Torre de la Corona. T'ho juro!<wait>
```
Comment (kusfo) [inline psrp/ca/script.yaml:1412]: Here, they speak like Majorcan (in theory)

File: psrp/ca/script.yaml [hunk 2]
```diff
@@ -1374,64 +1374,64 @@
...
 
 - offsets: 52b2
   # Dishonest Dezorian
   # <number> is always 10.
-  ca: Hi ha un passatge secret al desè nivell de sa caverna de Dezoris. No t'ensarrono!<wait>
+  ca: Hi ha un passatge secret al desè nivell de la caverna de Dezoris. No t'ensarrono!<wait>
```
Comment (kusfo) [inline psrp/ca/script.yaml:1417]: Same here

File: psrp/ca/script.yaml [hunk 3]
```diff
@@ -1374,64 +1374,64 @@
...
+  ca: Es fruits de Laerma són blaus, i tenyeixen la roba.<wait>
 
 - offsets: 52be
   # Dishonest Dezorian
-  ca: Pots emprar un cristall per fer créixer fruit a un arbre de Laerma.<wait>
+  ca: Pots emprar un cristall per fer créixer fruits a un arbre de Laerma.<wait>
 
 - offsets: 52c4
   # Dishonest Dezorian
-  ca: Tots es de Palma son benvenguts aquí. De veritat!<wait>
```
Comment (kusfo) [inline psrp/ca/script.yaml:1429]: Same here

File: psrp/ca/script.yaml [hunk 4]
```diff
@@ -1374,64 +1374,64 @@
...
+  ca: Tots els de Palma son benvinguts aquí. De veritat!<wait>
 
 - offsets: DishonestDezorianFix
   # Dishonest Dezorian
   # We add a script entry here. The original game accidentally uses an entry from Sopia.
-  ca: Es del poble de la vora són uns entabanadors.<wait>
+  ca: Els del poble de la vora són uns entabanadors.<wait>
```
Comment (kusfo) [inline psrp/ca/script.yaml:1434]: Same here

File: psrp/ca/script.yaml [hunk 5]
```diff
@@ -1440,7 +1440,7 @@
 - offsets: 5815
   # Dungeon Southwest of Corona Tower
   # Dishonest Dezorian
-  ca: Ves amb compte més endavant. Gira a sa esquerra a la bifurcació.<wait>
```
Comment (kusfo) [inline psrp/ca/script.yaml:1443]: I would maintain the "sa" as in the previous phrases

**Review by kusfo — COMMENTED** [review]
In general, I think you did a great job correctign my typos and "barbarisms". The only point I'll mostly maintain as it is, is regarding the two Dezorian populations that speak with Lleida and Majorca accent (Although maybe I wasn't really successful):

**Comment by RetroAND** [comment]
Sure, you are the original translator here after all.
I will be happy with whathever decision you take in this regard.
The only thing I noticed is about the cave of "Naula", where there is a cake shop... knowing they used words from other languages, especially Spanish, is there any chance they used the word "Neula" and just got distorted in the multiple translations? In my opinion that would make sense for two reasons: first that they can be bought in cake shops and bakeries, second because it has a tubular shape and could be reminiscent of a cave/tunnel. What do you think?

In any case, thank you very much for reviewing my changes.

File: psrp/ca/script.yaml [hunk 6]
```diff
@@ -2,33 +2,33 @@
...
-    Escolta'm amb atenció, el meu temps s'esgota.<wait more>
+    Escolta'm amb atenció, el meu temps s'acaba.<wait more>
     Els rumors que hem sentit són certs. Lashiec ha...canviat.<line>
     I va més enllà dels seus soldats controlant els carrers.<wait more>
     Tots els monstres a l'exterior, tota la foscor sobre nostre...<line><line>
     D'alguna manera, tot ens porta cap ell.<wait more>
     Vaig intentar descobrir el perquè, suposo que era a prop de la veritat.<line><line>
     Només ens queda una opció, pel bé de tots nosaltres.<wait more>
-    Hem d'aturar en Lashiec.<line><line>
+    Hem d'aturar-lo.<line><line>
```
[the remark this answers is not in the document]
Comment (RetroAND) [reply psrp/ca/script.yaml:22]: In this case we are already speaking about him, mentioning it again is redundant. Have in mind the character is dying and has to choose the words carefully.

File: psrp/ca/script.yaml [hunk 7]
```diff
@@ -235,7 +235,7 @@
 - offsets: 4dc7
   ca: |
     Benvingut a Shion, el port més transitat de Palma... o almenys ho era.<wait more>
-    Tal com va el món últimament, ja no hi ha gaire comerç.<wait>
+    En l'estat actual ja no hi ha gaire comerç.<wait>
```
[the remark this answers is not in the document]
Comment (RetroAND) [reply psrp/ca/script.yaml:238]: No problem

File: psrp/ca/script.yaml [hunk 8]
```diff
@@ -323,38 +323,38 @@
...
+  ca: Oh, d'acord.<wait>
 
 - offsets: 4d4f,4d60
   # Passport office (say "yes" to question about doing bad things)
   # Passport office (say "yes" to question about having an illness)
   ca: Ho sento, no compleix els requisits. Torni més endavant. Següent?<wait>
 
 - offsets: 5939
-  ca: Aquest és un vol sense parades a Motavia. Puja?<end>
+  ca: Aquest és un vol sense escales a Motavia. Ha de pujar?<end>
```
[the remark this answers is not in the document]
Comment (RetroAND) [reply psrp/ca/script.yaml:341]: Yes, and he is using the formal register (aka "parlar 'de vostè' "). I thought it was a better option than the informal register.

File: psrp/ca/script.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (kusfo) [reply psrp/ca/script.yaml:22]: Sounds good!

File: psrp/ca/script.yaml (hunk 8 above)
[the remark this answers is not in the document]
Comment (kusfo) [reply psrp/ca/script.yaml:341]: It looks to me live an overformal over the already formal "Puja?". Its informal form would be "Puges?"

**Comment by kusfo** [comment]
> Sure, you are the original translator here after all. I will be happy with whathever decision you take in this regard. The only thing I noticed is about the cave of "Naula", where there is a cake shop... knowing they used words from other languages, especially Spanish, is there any chance they used the word "Neula" and just got distorted in the multiple translations? In my opinion that would make sense for two reasons: first that they can be bought in cake shops and bakeries, second because it has a tubular shape and could be reminiscent of a cave/tunnel. What do you think?
> 
> EDIT: I would recommend placing translator notes in the parts where dialects and accents are employed in order to save them from further work by other contributors.
> 
> In any case, thank you very much for reviewing my changes.

About Naula cave, you can have a look at the original Japanese here: https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml. The original Kanji is ナウラ. I don't know Japanese, but Google Translate transliterates the name as "naura", but it gives no meaning (Seems a proper name). 

About the translation notes, they're indeed in the same file https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml , probably they weren't copied when the script was split per language. (Look at the Dezorian town notes)

**Comment by RetroAND** [comment]
> About Naula cave, you can have a look at the original Japanese here: https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml. The original Kanji is ナウラ. I don't know Japanese, but Google Translate transliterates the name as "naura", but it gives no meaning (Seems a proper name).
> 
> About the translation notes, they're indeed in the same file https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml , probably they weren't copied when the script was split per language. (Look at the Dezorian town notes)

I will add proper translation notes for the Catalan version, so further contributors don't mess like I did. :)

In respect to that bit of Japanese; nope, that's not kanji but katakana. Japanese have three different script types consisting in the kanji (ideograms) and two sylabaries. Katakana is the equivalent of italics in our scripts since it is reserved to imported words, neologisms and that kind of stuff. Kanas are phonetic, and are transcriptions of how they hear the word per se. For instance, "ice-cream" gets translated as 「アイスクリーム」(romaji: "aisukuriimu"). However conversion to Japanese is not lossless and a single misunderstanding can cause variations in the resulting transcription. It could differ so much than an actual retranslation of the word may not be possible anymore - as an example at the end of WWII a Japanese soldier guarding old swords gave a Masamune (I think it was) to a supposed American GI, wrote its name in Katakana and let him go, as a result authorities are still to this day trying to figure the name of the thief and the sword (classified as national treasure) is lost.

Sorry for the wall of text and the tale, but I think with some examples it is better to understand why I made the point, which is that if something in Japanese is written in katakana, it has no actual meaning and sounds like a word you know, chances are that it could be that word imported into Japanese. ;)

**Comment by kusfo** [comment]
> > About Naula cave, you can have a look at the original Japanese here: https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml. The original Kanji is ナウラ. I don't know Japanese, but Google Translate transliterates the name as "naura", but it gives no meaning (Seems a proper name).
> > About the translation notes, they're indeed in the same file https://github.com/maxim-zhao/psrp/blob/master/psrp/script.yaml , probably they weren't copied when the script was split per language. (Look at the Dezorian town notes)
> 
> I will add proper translation notes for the Catalan version, so further contributors don't mess like I did. :)
> 
> In respect to that bit of Japanese; nope, that's not kanji but katakana. Japanese have three different script types consisting in the kanji (ideograms) and two sylabaries. Katakana is the equivalent of italics in our scripts since it is reserved to imported words, neologisms and that kind of stuff. Kanas are phonetic, and are transcriptions of how they hear the word per se. For instance, "ice-cream" gets translated as 「アイスクリーム」(romaji: "aisukuriimu"). However conversion to Japanese is not lossless and a single misunderstanding can cause variations in the resulting transcription. It could differ so much than an actual retranslation of the word may not be possible anymore - as an example at the end of WWII a Japanese soldier guarding old swords gave a Masamune (I think it was) to a supposed American GI, wrote its name in Katakana and let him go, as a result authorities are still to this day trying to figure the name of the thief and the sword (classified as national treasure) is lost.
> 
> Sorry for the wall of text and the tale, but I think with some examples it is better to understand why I made the point, which is that if something in Japanese is written in katakana, it has no actual meaning and sounds like a word you know, chances are that it could be that word imported into Japanese. ;)

I'm fine to convert the name to Neula, it'll give a "Catalan" feeling to the locations :-)

**Comment by kusfo** [comment]
Please, update with the final changes, and we can ask @maxim-zhao to approve! 😄

File: psrp/ca/script.yaml (hunk 8 above)
[the remark this answers is not in the document]
Comment (RetroAND) [reply psrp/ca/script.yaml:341]: What about "pujaria?" not as formal as "pugen?" and more expressive than "puja?"

File: psrp/ca/script.yaml [hunk 9]
```diff
@@ -1374,64 +1374,64 @@
...
 
 - offsets: 527a
   # Honest Dezorian
-  ca: La Torre de Corona és a lo altre costat de les montanyes del nord.<wait>
+  ca: La Torre de Corona és a l'altre costat de les muntanyes del nord.<wait>
 
 - offsets: 5280
   # Honest Dezorian
-  ca: La cova de Dezoris és a ponent de la Torre de Corona. Digues-li hola a los nostres amics allà!<wait>
+  ca: La cova de Dezoris és a l'oest de la Torre de Corona. Digues-li hola als nostres amics allà!<wait>
```
[the remark this answers is not in the document]
Comment (kusfo) [reply psrp/ca/script.yaml:1385]: Sure!

File: psrp/ca/script.yaml (hunk 8 above)
[the remark this answers is not in the document]
Comment (kusfo) [reply psrp/ca/script.yaml:341]: Sounds good!

**Comment by RetroAND** [comment]
Sorry for the delay, real life takes its toll on my time.

I am not fluent with Majorcan, I don't know many of the specialized words in that dialect. If you could drop my changes on the Dezorians I would be happy with that. Also, I think it would be a good idea to make all Dezorians speak the same dialect - I don't like that much the idea of having only liars to speak Majorcan... I mean, I know a Majorcan and he is a liar and a troublemaker, but that doesn't mean all of them are. For this reason I would like to ask if you could revise this section.

What do you think about it?

Thank you very much

**Comment by kusfo** [comment]
> Sorry for the delay, real life takes its toll on my time.
> 
> I am not fluent with Majorcan, I don't know many of the specialized words in that dialect. If you could drop my changes on the Dezorians I would be happy with that. Also, I think it would be a good idea to make all Dezorians speak the same dialect - I don't like that much the idea of having only liars to speak Majorcan... I mean, I know a Majorcan and he is a liar and a troublemaker, but that doesn't mean all of them are. For this reason I would like to ask if you could revise this section.
> 
> What do you think about it?
> 
> Thank you very much

About them speaking the same dialect, the purpose in the Japanese original (where they spoke in two different regional dialects), was to know who was telling truth or lie. I didn't choose Majorcan for anything, just because with the very few phrases, I choose Lleidatà and Majorcan for the Dezorians, as they're two of the dialects that are more different (in the Spanish translation, I used Murciando and Galician spanish).

**Comment by RetroAND** [comment]
> > Sorry for the delay, real life takes its toll on my time.
> > I am not fluent with Majorcan, I don't know many of the specialized words in that dialect. If you could drop my changes on the Dezorians I would be happy with that. Also, I think it would be a good idea to make all Dezorians speak the same dialect - I don't like that much the idea of having only liars to speak Majorcan... I mean, I know a Majorcan and he is a liar and a troublemaker, but that doesn't mean all of them are. For this reason I would like to ask if you could revise this section.
> > What do you think about it?
> > Thank you very much
> 
> About them speaking the same dialect, the purpose in the Japanese original (where they spoke in two different regional dialects), was to know who was telling truth or lie. I didn't choose Majorcan for anything, just because with the very few phrases, I choose Lleidatà and Majorcan for the Dezorians, as they're two of the dialects that are more different (in the Spanish translation, I used Murciando and Galician spanish).

It is both understandable and logical, and that sense just an afterthought. What would be the fastest way to restore the Majorcan strings? Because it is the last thing remaining... is it correct?

**Comment by maxim-zhao** [comment]
Is this ready to merge? I guess from the above there's still some work to do.

**Comment by RetroAND** [comment]
Hello Maxim,

I forgot to check it for a long time. If you allow me for a final revision tonight tomorrow it could be merged.

Thank you in advance

**Comment by maxim-zhao** [comment]
There's no hurry, take your time.

File: psrp/ca/script.yaml (hunk 4 above)
Comment (RetroAND) [reply psrp/ca/script.yaml:1434]: Done

File: psrp/ca/script.yaml (hunk 9 above)
[the remark this answers is not in the document]
Comment (RetroAND) [reply psrp/ca/script.yaml:1385]: You are right, if everybody talks the same dialect, difficulty on this version would increase for no apparent reason. I will remove comments on the honest Dezorians and leave it as it is for now.

Fortunately Catalan has other dialects and if needed we can use Valencian, Aragonese, Andorran or even the most exotic among them, from Alguer.

**Comment by RetroAND** [comment]
I dug a little into Majorcan and now dishonest Dezorians could pass as native. I used more dialectal words than in the version I overwrote by accident. Still, if you could find a real Majorcan to check it it would be great - I am fine with my dialect and variant but I am not a replacement for a native Majorcan.

I don't know how to generate the patch and applying it, therefore I cannot test my changes for an in-depth quality check.

Finally, thank you for reminding me about this - I am always busy with something and sometimes I forget.

**Comment by RetroAND** [comment]
In principle, there is no more changes for now; I think it is ready for the pull.
I only hope that it is useful.

**Comment by maxim-zhao** [comment]
Many thanks for your attention. I could not make this project work without the help of the native speakers and editors to localise the text, or those who have contributed so much to the translation itself. I usually do releases around the anniversary of the game's release (20th December) so I have plenty of time to get this in 😄

**Pull request merged**